sdk/cosmos/azure-cosmos-spark_3-2_2-12/pom.xml

```diff
@@ -105,11 +105,6 @@
       <version>3.2.0</version> <!-- {x-version-update;cosmos-spark_3-2_org.apache.spark:spark-hive_2.12;external_dependency} -->
       <scope>test</scope>
     </dependency>
-    <dependency>
```

Comment:
Dependency is declared in the parent POM, resulting in the dependency being declared twice, removing this one